### ISSUE TITLE
Lura Toon Blocked

### ISSUE THREAD

**Issue by PeroIF** [keiyoushi/tachiyomi-extensions-source#2705]
### Source information

Lura Toon 1.4.43

### Source language

Português (Brasil)

### Steps to reproduce

1. Open source
2. Open anything
3. Open a chapter

### Expected behavior

![Screenshot_20240501_091341_Tachiyomi](https://github.com/keiyoushi/extensions-source/assets/92130026/508c8b59-d7a5-4bf8-b1c0-9c0574ddb452)
Should open the chapter

### Actual behavior

![Screenshot_20240501_091526_Tachiyomi](https://github.com/keiyoushi/extensions-source/assets/92130026/e841e394-776c-43e4-aea4-7bc8caa6f2c5)
![Screenshot_20240501_091538_Tachiyomi](https://github.com/keiyoushi/extensions-source/assets/92130026/40a3b246-504c-42f9-b08f-3945e0fff52d)
It's saying I have been blocked by Cloudfrare (I don't know if other people have been blocked, but this same message appears in other 2 extensions)

### Mihon/Tachiyomi version

0.15.3

### Android version

Android 14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by PeroIF** [comment]
The others that appear this message are:
  Cerise Scan 1.4.44
  Wicked Witch Scan 1.4.38

**Comment by nomaxsnx** [comment]
![Screenshot_2024-05-01-16-26-19-457_app mihon](https://github.com/keiyoushi/extensions-source/assets/157644160/12148d48-614c-405d-b62b-39d0d3af006c)

Extension lura toon ok

**Comment by nomaxsnx** [comment]
![Screenshot_2024-05-01-16-27-03-520_app mihon](https://github.com/keiyoushi/extensions-source/assets/157644160/2c2960b8-5dc4-4ac6-b225-46534578a2cd)


No load imagem

**Comment by nomaxsnx** [comment]
![Screenshot_2024-05-01-16-22-52-577_app mihon](https://github.com/keiyoushi/extensions-source/assets/157644160/938a3553-9625-413d-925f-acebda879d42)


No download image

**Comment by nomaxsnx** [comment]
> The others that appear this message are: Cerise Scan 1.4.44 Wicked Witch Scan 1.4.38

Recomendo que troca o User-agents 

Use esse: Mozilla/5.0 (Linux; Android 10; HD1913) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.6261.119 Mobile Safari/537.36 EdgA/122.0.2365.76

Mesmo assim não irá carregar a imagem 

DNS sobre HTTPS
Cloudfare

**Comment by paz-web** [comment]
Fiz isso e não adiantou mais alguma coisa que eu posso fazer?

**Comment by nomaxsnx** [comment]
> Fiz isso e não adiantou mais alguma coisa que eu posso fazer?

Espera a resposta dos desenvolvedores da extensão se tem como corrigir ou não.

**Comment by JosephShay** [comment]
Hello Brazilian friends, I had the same problem in an issue and talked to one of the admins on discord.
the conclusion was that the lura toon extension is no longer running with devices below 6gb of ram due to the weight it has to run

the website serves zips files instead of images, so the extension has some complicated logic to download, unzip, decode, and finally merge the images to return to the app, all in ram. (too low ram and/or not powerful enough cpu/gpu)

Of course, it's probably not that, but it's an alternative to a problem that I had with the same extension.

_**my device is currently a realme 7i.**_

**Comment by nomaxsnx** [comment]
> Hello Brazilian friends, I had the same problem in an issue and talked to one of the admins on discord. the conclusion was that the lura toon extension is no longer running with devices below 6gb of ram due to the weight it has to run
> 
> the website serves zips files instead of images, so the extension has some complicated logic to download, unzip, decode, and finally merge the images to return to the app, all in ram. (too low ram and/or not powerful enough cpu/gpu)
> 
> Of course, it's probably not that, but it's an alternative to a problem that I had with the same extension.
> 
> _**my device is currently a realme 7i.**_

ohh, good afternoon that's normal because of the big colors on the mangas, manhua and  etc.

as the site uses the theme: "PeachScan".


    /**
     * ActivityManager#isLowRamDevice is based on a system property, which isn't
     * necessarily trustworthy. 1GB is supposedly the regular threshold.
     *
     * Instead, we consider anything with less than 3GB of RAM as low memory
     * considering how heavy image processing can be.
     */
    private val isLowRamDevice by lazy {
        val ctx = Injekt.get<Application>()
        val activityManager = ctx.getSystemService("activity") as ActivityManager
        val memInfo = ActivityManager.MemoryInfo()

        activityManager.getMemoryInfo(memInfo)

        memInfo.totalMem < 3L * 1024 * 1024 * 1024
    }


you have to increase the code's memory, because it's limited to 3GB.

If you increase it, you'll have to limit the number of mobile devices available.

mine is: "Poco X4 GT".

**Comment by nomaxsnx** [comment]
![image](https://github.com/keiyoushi/extensions-source/assets/157644160/8d104c70-d55f-4c26-b3c2-54ee9111079d)


I just saw that after downloading the image it comes in: "txt", apparently it is encrypted

since it doesn't make sense, kkkkk, since those who won't buy the license to do it, I just want to screw with the extension.

**Comment by nomaxsnx** [comment]
![image](https://github.com/keiyoushi/extensions-source/assets/157644160/1c5c612c-ca78-4744-a277-1c89698e3906)

as I don't know much about html and cryptography, I think it's because of the "blob", which isn't working.

**Comment by JosephShay** [comment]
Well, your problem is different from the one I had.
unfortunately my maximum on my cell phone is 4gb by nature and there are no usage limitations. so I migrated to another extension unfortunately.

If they don't respond here, try discord.

**Comment by nomaxsnx** [comment]
> Well, your problem is different from the one I had. unfortunately my maximum on my cell phone is 4gb by nature and there are no usage limitations. so I migrated to another extension unfortunately.
> 
> If they don't respond here, try discord.

don't even try to talk on their discord, because of the "Businessman's nickname on the dc", which screws up the extension.

either developers update the extension or they turn it off.

because if they're going to increase the memory they'll have to limit it on mobile devices, so the old ones won't work, only the current ones.

**Comment by JosephShay** [comment]
Unfortunately this is not something that keiyoushi can solve, if there is, as it is a lot of work for such a small team or less, in itself it is only a problem because of the scan website itself which is quite heavy due to changes in it.

You can still try to go to discord and talk to the people there, in the case of keiyoushi itself and not scan (they don't like tachiyomi users)

**Comment by nomaxsnx** [comment]
> Unfortunately this is not something that keiyoushi can solve, if there is, as it is a lot of work for such a small team or less, in itself it is only a problem because of the scan website itself which is quite heavy due to changes in it.
> 
> You can still try to go to discord and talk to the people there, in the case of keiyoushi itself and not scan (they don't like tachiyomi users)

I know, I almost got banned from the discord of: "LuraToon".

The only thing I can do is wait for an update or let them know that they're going to remove the extension.

They don't like the extension because their processors are screwed up (since it's normal).

**Comment by JosephShay** [comment]
the message from the lura team was for you to change the browser that is no longer compatible, currently they said that chrome is the best alternative.
try using the chrome domain within tachiyomi and share with us.

**Comment by nomaxsnx** [comment]
> the message from the lura team was for you to change the browser that is no longer compatible, currently they said that chrome is the best alternative.
> try using the chrome domain within tachiyomi and share with us.

I ask you my friend: "how do I change the domain"?

**Comment by AwkwardPeak7** [comment]
this is not the place to have a nice long chat
